integration-cli/docker_cli_ps_test.go

```diff
@@ -200,10 +201,19 @@ func (s *DockerSuite) TestPsListContainersFilterStatus(c *check.C) {
 	c.Assert(RemoveOutputForExistingElements(containerOut, existingContainers), checker.Equals, secondID)
 
 	result := cli.Docker(cli.Args("ps", "-a", "-q", "--filter=status=rubbish"), cli.WithTimeout(time.Second*60))
-	result.Assert(c, icmd.Expected{
-		ExitCode: 1,
-		Err:      "Invalid filter 'status=rubbish'",
-	})
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+
+		result.Assert(c, icmd.Expected{
+			ExitCode: 1,
+			Err:      "Unrecognised filter value for status: rubbish",
```

Comment:
We could have only one assertion and just check and have the `Err` in a variable :angel:

Reply:
👍